module/Decision/src/Decision/Controller/FileBrowser/FileNode.php

```diff
@@ -64,4 +72,61 @@ public function getName()
+            'fa-file-csv' => array('csv'), //Only available as solid icon in free version
+            'fa-file-pdf' => array('pdf', 'pdfa'),
+            'fa-file-video' => array('webm', 'mpg', 'mp2', 'mpeg', 'mpe', 'mpv', 'ogg', 'mp4', 'm4p', 'm4v', 'avi',
+                                     'wmv', 'mov', 'qt', 'flv', 'swf', 'avchd', 'h264', 'mpeg4'),
+            'fa-file-audio' => array('3gp', 'aa', 'aac', 'aax', 'act', 'aiff', 'alac', 'amr', 'ape', 'au', 'awb', 'dct',
+                                     'dss', 'dvf', 'flac', 'gsm', 'iklax', 'ivs', 'm4a', 'm4b', 'm4p', 'mmf', 'mp3',
+                                     'mpc', 'msv', 'nmf', 'nsf', 'ogg, .oga, .mogg', 'opus', 'ra, .rm', 'raw', 'sln',
+                                     'tta', 'voc', 'vox', 'wav', 'wma', 'wv', 'webm', '8svx'),
+            'fa-file-image' => array('ani', 'anim', 'apng', 'art', 'bmp', 'bpg', 'bsave', 'cal', 'cin', 'cpc', 'cpt',
+                                     'dds', 'dpx', 'ecw', 'exr', 'fits', 'flic', 'flif', 'fpx', 'gif', 'hdri', 'hevc',
+                                     'icer', 'icns', 'ico / cur', 'ics', 'ilbm', 'jbig', 'jbig2', 'jng', 'jpeg',
+                                     'jpeg-ls', 'jpeg 2000', 'jpeg xr', 'jpeg xt ', 'jpeg-hdr', 'kra', 'mng', 'miff',
+                                     'nrrd', 'pam', 'pbm / pgm / ppm / pnm', 'pcx', 'pgf', 'pictor', 'png', 'psd',
+                                     'psb', 'psp', 'qtvr', 'ras', 'rgbe ', 'logluv tiff', 'sgi', 'tga', 'tiff',
+                                     'tiff/ep', 'tiff/it', 'ufo/ ufp', 'wbmp', 'webp', 'xbm', 'xcf', 'xpm', 'xwd'),
+            'fa-file-archive' => array('.a', 'ar', 'cpio', 'shar', 'LBR', 'iso', 'lbr', 'mar', 'sbx', 'tar', '7z',
+                                       's7z', 'ace', 'afa', 'alz', 'apk', 'ar', 'ark', 'arc', 'cdx', 'arj', 'b1', 'b6z',
+                                       'ba', 'bh', 'cab', 'car', 'cfs', 'cpt', 'dar', 'dd', 'dgc', 'dmg', 'ear', 'gca',
+                                       'ha', 'hki', 'ice', 'jar', 'kgb', 'lzh', 'lha', 'lzx', 'pak', 'partimg', 'paq6',
+                                       'paq7', 'paq8', 'pea', 'pim', 'pit', 'qda', 'rar', 'rk', 'sda', 'sea', 'sen',
+                                       'sfx', 'shk', 'sit', 'sitx', 'sqx', 'tar.gz', 'tgz', 'tar.Z', 'tar.bz2', 'tbz2',
+                                       'tar.lzma', 'tlz', 'tar.xz', 'txz', 'uc', 'uc0', 'uc2', 'ucn', 'ur2', 'ue2',
+                                       'uca', 'uha', 'war', 'wim', 'xar', 'xp3', 'yz1', 'zip', 'zipx', 'zoo', 'zpaq',
+                                       'zz'),
+            'fa-file-alt' => array('txt', 'tex', 'text', 'ini', 'md'),
+            'fa-copy' => array('bak', 'tmp', 'dmp'),
+        );
+
+        foreach ($types as $type => $extensions)
+        {
+            if (in_array($this->getExtension(), $extensions)) return $type;
+        }
```

Comment:
This also isn't up to PSR-1. Basically all changes in this file aren't.

Also, the `$types` variable here, should probably be declared as a private class member, not inline in the function.

Reply:
Has been changed in 3eea7fd4ced3cb3f1bcdaa008fd431af0f57ebfa